app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :post
+
+  def update_comments_counter(post)
+    post.comments_counter = Comment.where(post_id: post.id).count
+  end
```

Comment:
[OPTIONAL]
- You could use [rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) to call this method every time a new post is saved to the database. After that, you can make the method private. It can look like this
```
  after_save :update_comments_counter

  private

  def update_comments_counter(post)
    post.comments_counter = Comment.where(post_id: post.id).count
  end
```